sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md

```diff
@@ -141,6 +141,19 @@ Use with [DwmSetWindowAttribute](/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindo
 
 The maximum recognized <b>DWMWINDOWATTRIBUTE</b> value, used for validation purposes.
 
+### -field DWMWA_USE_IMMERSIVE_DARK_MODE:20
```

Comment:
I thought I already documented this (and the old value of 19) before. Also It is supported before Windows 11 I thought.

I did but someone reverted it.

See also: https://github.com/MicrosoftDocs/sdk-api/pull/966